File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/ItemList/Slist.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Slist {

    @SerializedName("field")
    @Expose
    private String field;
    @SerializedName("name")
    @Expose
    private String name;

    public String getField() {
        return field;
    }

    public void setField(String field) {
        this.field = field;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itemlist/Slist.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Slist {

    @SerializedName("field")
    @Expose
    var field: String? = null
    @SerializedName("name")
    @Expose
    var name: String? = null

}
